index.html

```diff
@@ -197,9 +198,15 @@ <h3>Working Group Notes, registries etc</h3>
         <li><a href="https://github.com/w3c/png-hdr-pq/pulse">PQ HDR in PNG</a></li>
         <li><a href="https://github.com/w3c/tt-profile-registry/pulse">TTML Profile Registry</a></li>
         <li><a href="https://github.com/w3c/ttml-webvtt-mapping/pulse">TTML to WebVTT mapping</a></li>
+        <li><a href="https://github.com/w3c/dapt-reqs">Requirements on Dubbing and Audio description Profiles of TTML2 (DAPT)</a></li>
+      </ul>
+      <h3>Suspended Recommendation Track specification repositories</h3>
```

Comment:
Suspended doesn't have a meaning in the Process, I think, so this is okay, but I guess we should decide what to do with them at some point.

Reply:
Yes...
If once we have published as FPWD, we may write with using Process terms, but neither ADPT nor TTML3 have passed.
https://www.w3.org/2023/Process-20230612/#tr-end

Reply:
(please mark this as resolved if you are ok with current text)